### ISSUE TITLE
consider integrating fix-ev

### ISSUE THREAD

**Issue by juliangruber** [defunctzombie/dom-events#4]
https://github.com/juliangruber/fix-ev

for older browsers

**Comment by juliangruber** [comment]
if you want this in i'll make a pull request

**Comment by defunctzombie** [comment]
Yes please. I would like to see a PR for it to get an idea of what that would look like.

Tests included would be :heart:

**Comment by callumacrae** [comment]
Would you still be interested in a PR for this?

**Comment by defunctzombie** [comment]
Couldn't hurt. I personally don't have use case for old IE but if you do I would entertain the PR.

**Comment by callumacrae** [comment]
I do :cry: 

I'm not actually using your library, because I'm only interested in `.on`. Adding fix-ev to your library isn't trivial—if you want `.off` to work correctly in oldIE still, you have to store every listener in an array and remove the correct one.

If I need to use more than `.on`, I'll use this library and add this functionality :smile:
